src/components/ChatList/ChannelItem.style.ts

```diff
@@ -1,22 +1,22 @@
 import {StyleSheet} from 'react-native';
 
 import dimen from '../../utils/dimen';
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
 
 export const channelItemStyles = StyleSheet.create({
   channelContainer: {
     display: 'flex',
     flexDirection: 'row',
     paddingLeft: dimen.normalizeDimen(20),
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property in the `channelContainer` style has been updated to use `COLORS.white` instead of `colors.white`. This change aligns with the PR's objective of using the standardized `COLORS` object for color values.
```diff
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->